frontends/web/src/routes/settings/components/appearance/activeCurrenciesDropdownSetting.tsx

```diff
@@ -14,14 +14,16 @@
  * limitations under the License.
  */
 
-import { store, SharedProps, formattedCurrencies } from '../../../../components/rates/rates';
+import { formattedCurrencies } from '../../../../components/rates/rates';
 import { SettingsItem } from '../settingsItem/settingsItem';
-import { share } from '../../../../decorators/share';
 import { ActiveCurrenciesDropdown } from '../dropdowns/activecurrenciesdropdown';
 import { useTranslation } from 'react-i18next';
+import { useContext } from 'react';
```

Comment:
since you are here, please move imports from react first, then npm such as 'react-i18nex' then ours.